build.gradle

```diff
@@ -620,6 +620,7 @@ defineProperty("HUDSON_JOB_NAME", "not_hudson")
 defineProperty("HUDSON_BUILD_NUMBER","0000")
 defineProperty("PROMOTED_BUILD_NUMBER", "0")
 defineProperty("MILESTONE_FCS", "false")
+defineProperty("WEBVIEW_VERSION", "18.4")
```

Comment:
Can you separate this from the previous 4 "Husdon" definitions by a blank line and also add a comment?